src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -108,5 +113,32 @@ private static string LoadRules()
                     Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
                     "Rules/BuiltInRules.json"));
         }
+
+        /// <summary>
+        /// Loads the configurations file from default directory if no file was passed as an optional parameter.
+        /// </summary>
+        /// <param name="configurations">Optional Command-line Configurations paramater.</param>
+        /// <returns>Configurations file path if exists.</returns>
+        private string LoadConfiguration(string configurations)
+        {
+            try
+            {
+                if (configurations == null)
+                {
+                    var defaultPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) +
+                        "/Configurations/Configuration.json";
```

Comment:
It might be necessary to use something like https://docs.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-6.0 here, for the Linux tests to pass (two are failing now)